system-addon/lib/ActivityStreamStorage.jsm

```diff
@@ -4,15 +4,15 @@ this.ActivityStreamStorage = class ActivityStreamStorage {
   /**
    * @param storeNames Array of strings used to create all the required stores
    */
-  constructor(options = {}) {
-    if (!options.storeNames || !options.telemetry) {
-      throw new Error(`storeNames and telemetry are required, called only with ${Object.keys(options)}`);
+  constructor({storeNames, telemetry}) {
```

Comment:
won't your request wrapper throw if telemetry is falsey?

Reply:
It is only used if non falsey.

https://github.com/mozilla/activity-stream/blob/106c699eed2f594dc280806d684ca2fdf3389f94/system-addon/lib/ActivityStreamStorage.jsm#L73